docs/displaysystem/basicdisplaysystem.md

```diff
@@ -61,3 +61,56 @@ If table contents start with `%html`, it is interpreted as an HTML.
 <img src="../assets/themes/zeppelin/img/screenshots/display_table_html.png" />
 
 > **Note :** Display system is backend independent.
+
+## Network
+
+With the `%network` directive, Zeppelin treats your output as a graph. Zeppelin can leverage the Property Graph Model.
+
+### What is the Labelled Property Graph Model?
+
+A [Property Graph](https://github.com/tinkerpop/gremlin/wiki/Defining-a-Property-Graph) is a graph that has these elements:
+
+* a set of vertices
+    * each vertex has a unique identifier.
+    * each vertex has a set of outgoing edges.
+    * each vertex has a set of incoming edges.
+    * each vertex has a collection of properties defined by a map from key to value
+* a set of edges
+    * each edge has a unique identifier.
+    * each edge has an outgoing tail vertex.
+    * each edge has an incoming head vertex.
+    * each edge has a label that denotes the type of relationship between its two vertices.
+    * each edge has a collection of properties defined by a map from key to value.
+
+<img src="https://github.com/tinkerpop/gremlin/raw/master/doc/images/graph-example-1.jpg" />
```

Comment:
@AhyoungRyu sorry i had lost your comment, no there is no special reason, if you think is better I can import the image inside the project